CONTRIBUTING.md

```diff
@@ -87,6 +87,7 @@ We capitalize the following terms:
 We do not capitalize the following terms:
 
 * content view
+* lifecycle environment
```

Comment:
Oh, we don't? :smile:  Seems we have lots of updates to do...

Reply:
You're somewhat correct. Maybe I drop it in this PR and create a follow-up PR to fix this:

````
$ rg "Lifecycle Environment" | wc
     28     363    4598
$ rg "lifecycle environment" | wc
    122    2081   22833
$ rg "Lifecycle environment" | wc
      6      58     838
````